src/Field/NominatimSearch/NominatimSearch.spec.jsx

```diff
@@ -113,16 +125,20 @@ describe('<NominatimSearch />', () => {
         dataSource: dataSource
       });
       wrapper.instance().onMenuItemSelected('752526');
-      expect(selectSpy.calledOnce).toBeTruthy();
-      expect(selectSpy.calledWith(dataSource[0], map)).toBeTruthy();
+      expect(selectSpy).toHaveBeenCalled();
+      expect(selectSpy.mock.calls[0][0]).toBe(dataSource[0]);
+      expect(selectSpy.mock.calls[0][1]).toBe(map);
```

Comment:
We might better use `.toHaveBeenCalledWith`  https://facebook.github.io/jest/docs/en/expect.html#tohavebeencalledwith